app-modules/project/src/Filament/Resources/ProjectResource.php

```diff
@@ -61,7 +60,7 @@ public static function canAccess(): bool
         /** @var User $user */
         $user = auth()->user();
 
-        return ProjectPageFeature::active() && $user->can('project.view-any');
+        return $user->can('project.view-any');
     }
```

Comment:
I think it actually would just be fine in this file to just remove this override of `canAccess` since the parent would be checking that same permission anyways.